krun/scheduler.py

```diff
@@ -384,37 +384,35 @@ def __init__(self, config, mailer, platform, dry_run=False):
         self.dry_run = dry_run
         self.log_path = self.config.log_filename(resume=True)
         self.manifest = ManifestManager(config, platform)
-        self.results = None
```

Comment:
I think it might be good to replace this with a comment to explain why we don't do the obvious thing (just in case we forget and try to 'simplify' the code later on).